src/odemis/driver/autoscript_client.py

```diff
@@ -1748,13 +1763,20 @@ def __init__(self, name: str, role: str, parent: SEM, rng: Optional[Dict[str, Tu
             "rz": model.Axis(unit=stage_info["unit"]["r"], range=rng["rz"]),
         }
 
+        # When raw coordinate system is selected, in theory just z should change
```

Comment:
Out of curiosity: why didn't you implement `_switch_coordinate_system` for the autoscript_client?

Reply:
There's no use case for using linked coordinate system now